.changeset/pretty-drinks-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend': minor
+---
+
+Scaffolder: Add 'eq' handlebar helper
```

Comment:
Could use a slightly more verbose explanation here, like:

--- 

Added an `eq` helper for use in software template YAML files. This can be used to execute a step depending on the value of an input, e.g.:

```yaml
steps:
  id: 'conditional-step'
  action: 'custom-action'
  if: '{{ eq parameters.myvalue "custom" }}',
```

Reply:
Sure, I've updated the explanation with yours 😄